src/configurations/destinations/salesforce_bulk_upload/db-config.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "SALESFORCE_BULK_UPLOAD",
+  "displayName": "Salesforce Bulk Upload",
+  "config": {
+    "supportsVisualMapper": true,
+    "auth": {
+      "type": "OAuth"
+    },
+    "disableJsonMapper": true,
+    "syncBehaviours": ["upsert"],
+    "transformAtV1": "none",
+    "saveDestinationResponse": true,
+    "includeKeys": ["oneTrustCookieCategories", "ketchConsentPurposes", "consentManagement"],
+    "excludeKeys": [],
+    "supportedAccountDefinitions": {
+      "rudderAccountId": ["DESTINATION_SALESFORCE_BULK_UPLOAD_OAUTH"]
+    },
+    "supportedSourceTypes": ["warehouse"],
+    "supportedMessageTypes": {
+      "cloud": ["identify"]
+    },
+    "destConfig": {
+      "defaultConfig": ["rudderAccountId"],
+      "warehouse": [
+        "connectionMode",
+        "oneTrustCookieCategories",
+        "ketchConsentPurposes",
+        "consentManagement"
+      ]
+    },
+    "secretKeys": [],
+    "supportedConnectionModes": {
+      "warehouse": ["cloud"]
+    }
+  },
+  "options": {
+    "isBeta": true
+  }
```

Comment:
Should we release it under a feature flag? Once fullScript starts using it, and everything looks good, we can enable it for all. @krishna2020